boards/arm/adafruit_feather_nrf52840/adafruit_feather_nrf52840_sense.yaml

```diff
@@ -0,0 +1,19 @@
+identifier: adafruit_feather_nrf52840_sense
```

Comment:
@kartben this identifier isn't picked up by `CONFIG_BOARD`, so hello_world would print "adafruit_feather_nrf52840" even when built against "adafruit_feather_nrf52840_sense". This isn't a huge problem, but is there another macro one could examine to determine in C which variant was built?

Reply:
Maybe this would be a job for `BOARD_REVISION`? However the `xiao_ble` doesn't use that

Reply:
I'd expect the `model` field from adafruit_feather_nrf52840_sense.dts to do the job, but I don't see it in devicetree_generated.h

Reply:
This identifier is not the one used at runtime, this is how you would typically do it https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/rpi_pico/Kconfig.defconfig#L6-L8. 
It looks like xiao_ble doesn't do things quite right (I am quite certain that hello world on XIAO BLE Sense would say "xiao_ble" and you might be better off looking at pico/picow.

Reply:
That did it; `strings build/zephyr/zephyr.bin | rg adafruit` now shows the _sense suffix. Thanks!